### PR TITLE
API-193: Coveralls code coverage

### PR DESCRIPTION
**Pull request by Arpit-Sharma-USC**
[JIRA](https://bynder.atlassian.net/browse/API-193)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,5 +1,8 @@
 # Bynder PHP SDK
 ![Build](https://github.com/Bynder/bynder-php-sdk/workflows/Build/badge.svg)
+[![Coverage Status](https://coveralls.io/repos/github/Bynder/bynder-php-sdk/badge.svg?branch=master)](https://coveralls.io/github/Bynder/bynder-php-sdk?branch=master)
+[![Packagist Version](https://img.shields.io/packagist/v/bynder/bynder-php-sdk)](https://packagist.org/packages/bynder/bynder-php-sdk)
```
Comment (betacar) [inline README.md:4]: Nice!

**Review by betacar — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (Arpit-Sharma-USC) [reply README.md:4]: muchas gracias

File: .github/workflows/build.yml [hunk 2]
```diff
@@ -8,18 +8,38 @@ on:
...
+          composer require --dev php-coveralls/php-coveralls
+
+      - name: Run tests
+        run: XDEBUG_MODE=coverage vendor/bin/phpunit tests --coverage-clover=build/logs/clover.xml
+
+      - name: Upload coverage results to Coveralls
+        env:
+          COVERALLS_REPO_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        run: |
+          composer global require php-coveralls/php-coveralls
```
Comment (githuib) [inline .github/workflows/build.yml:44]: Isn't this already done at line 35? (or shouldn't we remove that one?)

Comment (Arpit-Sharma-USC) [reply .github/workflows/build.yml]: yeah that's a dup. Fixed.

**Review by betacar — APPROVED** [review]
Approved

**Pull request merged**